kernel/src/schema/mod.rs

```diff
@@ -145,7 +145,7 @@ impl StructField {
         self
     }
 
-    pub fn get_config_value(&self, key: &ColumnMetadataKey) -> Option<&MetadataValue> {
+    pub fn get_metadata_value(&self, key: &ColumnMetadataKey) -> Option<&MetadataValue> {
```

Comment:
I like this change but (1) unsure if this should be here? and (2) this is breaking so let's make sure to call it out